src/test/resources/fixtures/lang/unnecessary-else.php

```diff
@@ -0,0 +1,75 @@
+<?php
```

Comment:
This definitely needs time for checking. 

What catches my attention is handling `elseif`: we need an option (disabled by default) to switch on/off `elseif` handling. When it's of if-statement containing `elseif`` should not be inspected at all.

Answering why: code with elseif branching smells as for me, therefore I'd give people illusions that they are improved it by fixing out remarks.